tests/foreman/ui/test_discoveryrule.py

```diff
@@ -194,7 +194,7 @@ def test_positive_list_host_based_on_rule_search_query(
 
     :BZ: 1731112
     """
-    ip_address = gen_ipaddr(ipv6=settings.server.is_ipv6)
+    ip_address = gen_ipaddr(ipv6=target_sat.network_type == NetworkType.IPV6)
```

Comment:
This test might fail as we generate ipv4 address here for dualstack run , but in the logic below with lookup we expect ipv6 address for dualstack too
```suggestion
    ip_address = gen_ipaddr(ipv6=target_sat.network_type in [NetworkType.IPV6, NetworkType.DUALSTACK])
```

Reply:
Or if https://github.com/SatelliteQE/robottelo/pull/17904#discussion_r2068365084 is implemented:
```suggestion
    ip_address = gen_ipaddr(ipv6=target_sat.network_type.has_ipv6)
```